underfs/s3/src/main/java/alluxio/underfs/s3/S3Utils.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.underfs.s3;
+
+import org.jets3t.service.acl.AccessControlList;
+import org.jets3t.service.acl.GrantAndPermission;
+import org.jets3t.service.acl.GroupGrantee;
+import org.jets3t.service.acl.Permission;
+
+/**
+ * Util functions for S3N under file system.
+ */
+public final class S3Utils {
+  /**
+   * Translates S3 bucket owner ACL to Alluxio owner mode.
+   *
+   * @param acl the acl of S3 bucket
+   * @param bucketOwnerId the bucket owner id
+   * @return the translated posix mode in short format
+   */
+  public static short translateBucketAcl(AccessControlList acl, String bucketOwnerId) {
```

Comment:
In my mind, utils classed are meant to be used to factor out code that is re-used in multiple places. I would argue that putting the translate bucket ACL logic into a utils class actually hurts readability because logic related to the S3UnderFileSystem implementation is fragmented across multiple classes. I don't feel strongly about this though, so it is fine to keep this as is.